### PR TITLE
Display actual FPS

### PR DESCRIPTION
**Pull request by AI0867**
That is, not calculated FPS, but the actual number of frames rendered during the previous second.

### CODE REVIEW

File: src/display.cpp [hunk 1]
```diff
@@ -1695,6 +1698,14 @@ void display::draw_wrap(bool update, bool force)
 		update_display();
 
 		frametimes_.push_back(SDL_GetTicks() - last_frame_finished_);
+		fps_counter_++;
+		const std::chrono::seconds current_second = std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now().time_since_epoch());
```
Comment (jyrkive) [inline src/display.cpp]: This line is too long. You can shorten it with

    using std::chrono::seconds;

File: src/display.cpp [hunk 2]
```diff
@@ -1695,6 +1698,14 @@ void display::draw_wrap(bool update, bool force)
...
+		if(current_second != fps_start_){
```
Comment (jyrkive) [inline src/display.cpp]: There should be a space between the closing parenthesis and the opening brace:

    if(current_second != fps_start_) {

File: src/display.cpp (hunk 1 above)
Comment (jyrkive) [inline src/display.cpp]: Please use `std::chrono::steady_clock` for this. `system_clock` is intended only for display.

File: src/display.cpp [hunk 3]
```diff
@@ -1695,6 +1698,14 @@ void display::draw_wrap(bool update, bool force)
...
+			std::cerr << "Second " << current_second.count() << ": FPS = " << fps_counter_ << '\n';
```
Comment (jyrkive) [inline src/display.cpp]: Please remove this debug print.

File: src/display.cpp [hunk 4]
```diff
@@ -1699,7 +1699,8 @@ void display::draw_wrap(bool update, bool force)
 
 		frametimes_.push_back(SDL_GetTicks() - last_frame_finished_);
 		fps_counter_++;
-		const std::chrono::seconds current_second = std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now().time_since_epoch());
+		using std::chrono::seconds;
+		const seconds current_second = std::chrono::duration_cast<seconds>(std::chrono::system_clock::now().time_since_epoch());
```
Comment (jyrkive) [inline src/display.cpp]: The line is still too long to fit in GitHub's view. Can you add a `using` for `duration_cast` as well?

**Comment by Vultraz** [comment]
Please merge if ready.

**Comment by jyrkive** [comment]
Merged. Thank you, @AI0867 ! :)

**Pull request merged**